scripts/build-webpack.js

```diff
@@ -58,7 +59,7 @@ const compilerLegacy = webpack({
         test: /\.js$/,
         exclude: /node_modules/,
         loader: 'babel-loader',
-        query: babelRc.env['pre-node5'],
+        query: babelRc.env.node4,
```

Comment:
Is it ok if we open another issue/pull request on that? This pr is becoming too big imho.